daliuge-engine/dlg/manager/proc_daemon.py

```diff
@@ -69,21 +69,24 @@ class DlgDaemon(RestServer):
     and the master manager (default: no) at creation time.
     """
 
-    def __init__(self, master=False, noNM=False, disable_zeroconf=False, verbosity=0):
+    def __init__(self, master=False, dim=False, nm=False, tm=False, disable_zeroconf=False, verbosity=0, daemonise=False, stop=False):
```

Comment:
Docstrings for each of this is necessary; I do not know what a `tm` is and it's important to have the parameters defined.

Reply:
Now I can see that it is `translator manager`. Please see comment elsewhere about this creating a dependency.